features/rhelah/atomic/rpm-ostree_bad_ca_cert.feature

```diff
@@ -0,0 +1,16 @@
+Feature: tests whether RH CA pinning works for rpm-ostree
+    It is critical for security that we're "pinned" to the RHN CA cert. If another server certificate is presented we should error out with something like Unacceptable TLS certificate
+
+Scenario: 0. Host provisioned and subscribed
+	When "all" host is auto-subscribed to "prod"
+	Then subscription status is ok on "all"
+and "1" entitlement is consumed on "all"
+
+Scenario: 1. Prepare mitmproxy
+	Given "git" are already installed on "rhel7"
+	and run command "command git clone https://github.com/ljozsa/tools.git" on "rhel7"
+	and run command "command /root/tools/mitmproxy_install.sh" on "rhel7"
+
+Scenario: 2. Direct all https traffic through mitmproxy which replaces original certs by certs signed by its own CA
+	When IP address of the "rhel7" is recorded
+	Then set proxy environment variable on the "rhelah" to the IP address of the "rhel7" host and attempt to run upgrade
```

Comment:
Ahh, right, you need to capture  the output of the command to check for the error, so that second step wouldn't directly work. Maybe instead have a step `check that an atomic host upgrade fails` which runs the command checks  the output.